packages/integration-tests/src/entities/LargePublisher.test.ts

```diff
@@ -0,0 +1,17 @@
+import { LargePublisher } from "./LargePublisher";
+import { newLargePublisher } from "./LargePublisher.factories";
+import { newPublisherGroup } from "./PublisherGroup.factories";
+import { newSmallPublisher } from "./SmallPublisher.factories";
+
+describe("LargePublisher", () => {
+  it.withCtx("can be find by inherited fields", async (ctx) => {
```

Comment:
Suggestion: Can we put this in `Inheritance.test.ts`?

We do have a fair amount of tests in like `Author.test.ts` or what not, but lately I've been trying to organize the tests by feature instead of by entity.

So like if we futz with something related to inheritance, we can easily rerun all of the inheritance-related tests.